src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,18 @@
+rootLogger.level = INFO
+appenders= console
+
+
+appender.console.type = Console
+appender.console.name = LogToConsole
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = %d{ABSOLUTE} %5p %c{1}:%L - %m%n
+rootLogger.appenderRef.stdout.ref = LogToConsole
+rootLogger.appenderRef.console.ref = LogToConsole
+
+### direct messages to file hibernate.log ###
```

Comment:
Looks legit but I have to ask... has this bit been manually tested?

Reply:
yes it has been tested manually. See comment below.